src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,120 @@
+- The `http.response_content_length` span data value is set to the [encoded body size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/encodedBodySize) of the resource.
+- The `http.decoded_response_content_length` span data value is set to the [decoded body size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/decodedBodySize) of the resource.
+- The `resource.render_blocking_status` span data value is set to the [render blocking status](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/renderBlockingStatus) of the resource.
+
+Here's an example snippet of creating a resource span manually with the SDK.
+
+```js
+// https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
+const resources = performance.getEntriesByType("resource");
+resources.forEach((entry) => {
+  const startTime = msToSec(entry.startTime);
+  const duration = msToSec(entry.duration);
+
+  const span = Sentry.startInactiveSpan({
+    startTimestamp: startTime,
+    op: entry.initiatorType
+      ? `resource.${entry.initiatorType}`
+      : "resource.other",
+    description: entry.name,
+    data: {
+      "http.response_transfer_size": entry.transferSize,
+      "http.response_content_length": entry.encodedBodySize,
+      "http.decoded_response_content_length": entry.decodedBodySize,
+      "resource.render_blocking_status": entry.renderBlockingStatus,
+    },
+  });
+
+  // override end timestamp to match performance entry
+  span.finish(startTime + duration);
+});
+```
+
+## Resources Page
+
+The **Resources** page gives you a quick overview of your application's resource performance for the selected project(s). You can use this page as a starting point to investigate potential problem resources and drill down to better understand how various resources are affecting your app's performance.
+
+Open the **Resources** page by clicking "Resources" in the sidebar, under "Performance". Alternatively, the **Performance** page also surfaces the highest impact queries in the [TBD] widget. From there, you can click "View All" to open the **Resources** page.
+
+At the top of the page, summary graphs for requests per minute (throughput) and average duration provide high-level insight into the performance of your resources. If you see an anomaly or want to investigate a time range further, you can click and drag to select a range directly in a graph to filter data for that time range.
+
+The resource table below shows a list of grouped resources, along with their type, their volume (requests per min), average duration, the total time your app spent loading that resource (time spent), and [average encoded size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/encodedBodySize) of the resource.
+
+Currently only JavaScript and CSS resources are supported.
```

Comment:
Consider moving this line up to the "Prerequisites and Limitations" section

Reply:
Good idea, Moved it!